internal/backend/crontab/crontab.go

```diff
@@ -0,0 +1,180 @@
+// Package crontab is a Temporal worker that implements recurring internal
+// maintenance tasks (e.g. cleanups, 3rd-party event watch renewals, etc.).
+//
+// In a normal state, Temporal will have: 1 available worker PER connected
+// AutoKitteh server, but ONLY 1 named schedule that triggers the main
+// workflow, which runs a few child workflows (1 per task), which run
+// multiple activities (1 per relevant connection).
+//
+// Child workflows run in parallel, so they don't block each other. However,
+// activities run sequentially per child workflow, so they won't overwhelm
+// 3rd-party services and constrained resources that they access.
+//
+// Note that even in the worst case (a child workflow fails or gets terminated
+// by the next invocation), successful activities won't need to run again, so
+// congestions will be resolved eventually no matter what.
+package crontab
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.temporal.io/api/enums/v1"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/workflow"
+	"go.uber.org/zap"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/configset"
+	"go.autokitteh.dev/autokitteh/internal/backend/temporalclient"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+)
+
+const (
+	taskQueueName = "internal-scheduler-task-queue"
+	workflowName  = "internal_maintenance"
+	scheduleID    = "internal_maintenance"
+)
+
+type Config struct {
+	Worker   temporalclient.WorkerConfig   `koanf:"worker"`
+	Workflow temporalclient.WorkflowConfig `koanf:"workflow"`
+	Activity temporalclient.ActivityConfig `koanf:"activity"`
+}
+
+var (
+	Configs = configset.Set[Config]{
+		Default: &Config{},
+	}
+
+	scheduleSpec = client.ScheduleSpec{
+		Intervals: []client.ScheduleIntervalSpec{{Every: 8 * time.Hour}},
```

Comment:
i think these should vary between handlers. we just need an abstraction on top of temporal, or something else.

Reply:
same as above, a nice big red colored TODO(ticket) is fine

Reply:
Added TODOs in `jira.go` and `google.com`